### ISSUE TITLE
build_app error: failed to read asset tags Images.xcassets

### ISSUE THREAD

**Issue by xe1x** [fastlane/fastlane#19445]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

Build_app action won't build for iOS app. The error message is error: `failed to read asset tags: The command (cd /Users/user/app/ios && /Applications/Xcode.app/Contents/Developer/usr/bin/actool --print-asset-tag-combinations --output-format xml1 /Users/user/app/ios/xxx/Images.xcassets)` exited with status 255.`

<details><pre>Completed error log</pre>
/Users//Library/Logs/gym/-.log
Command line invocation:
    /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -workspace .xcworkspace -scheme  -destination generic/platform=iOS -archivePath "/Users//Library/Developer/Xcode/Archives/2021-10-06/ 2021-10-06 17.46.07.xcarchive" clean archive CODE_SIGN_IDENTITY= CODE_SIGNING_REQUIRED=NO CODE_SIGN_ENTITLEMENTS= CODE_SIGNING_ALLOWED=NO

User defaults from command line:
    IDEArchivePathOverride = /Users//Library/Developer/Xcode/Archives/2021-10-06/ 2021-10-06 17.46.07.xcarchive
    IDEPackageSupportUseBuiltinSCM = YES

Build settings from command line:
    CODE_SIGN_ENTITLEMENTS = 
    CODE_SIGN_IDENTITY = 
    CODE_SIGNING_ALLOWED = NO
    CODE_SIGNING_REQUIRED = NO

note: Using new build system
note: Using codesigning identity override: 
note: Build preparation complete

** CLEAN SUCCEEDED **

note: Using new build system
note: Using codesigning identity override: 
note: Planning
Analyze workspace

Create build description
Build description signature: 4a9e75cac8d423fee8271d68df6b11f5
Build description path: /Users//Library/Developer/Xcode/DerivedData/-bhvqullpbywyvxhazlpppnjzxuug/Build/Intermediates.noindex/ArchiveIntermediates//IntermediateBuildFilesPath/XCBuildData/4a9e75cac8d423fee8271d68df6b11f5-desc.xcbuild

note: Build preparation complete
note: Building targets in parallel
error: failed to read asset tags: The command `(cd /Users//mobile-app/ios && /Applications/Xcode.app/Contents/Developer/usr/bin/actool --print-asset-tag-combinations --output-format xml1 /Users//mobile-app/ios//Images.xcassets)` exited with status 255. The command's standard error was:

2021-10-06 17:46:13.280 ibtoold[82042:778426] Requested but did not find extension point with identifier Xcode.InterfaceBuilderBuildSupport.PlatformDefinition for extension Xcode.IBCocoaTouchBuildSupport.PlatformDefinition.MacCatalyst of plug-in com.apple.dt.IDE.IBCocoaBuildSupport
2021-10-06 17:46:13.280 ibtoold[82042:778426] Requested but did not find extension point with identifier Xcode.InterfaceBuilderBuildSupport.PlatformDefinition for extension Xcode.IBCocoaBuildSupport.PlatformDefinition.Cocoa of plug-in com.apple.dt.IDE.IBCocoaBuildSupport
2021-10-06 17:46:13.281 ibtoold[82042:778426] Requested but did not find extension point with identifier Xcode.InterfaceBuilderBuildSupport.PlatformDefinition for extension Xcode.IBCocoaTouchBuildSupport.PlatformDefinition.CocoaTouch of plug-in com.apple.dt.IDE.IBCocoaTouchBuildSupport
2021-10-06 17:46:13.281 ibtoold[82042:778426] Requested but did not find extension point with identifier Xcode.InterfaceBuilderBuildSupport.PlatformDefinition for extension Xcode.IBAppleTVBuildSupport.PlatformDefinition.AppleTV of plug-in com.apple.dt.IDE.IBAppleTVBuildSupport
2021-10-06 17:46:13.282 ibtoold[82042:778426] Requested but did not find extension point with identifier Xcode.InterfaceBuilderBuildSupport.PlatformDefinition for extension Xcode.IDEInterfaceBuilder.PlatformDefinition.WatchOS of plug-in com.apple.dt.IDE.IDEInterfaceBuilderWatchKitBuildSupport
2021-10-06 17:46:13.299 ibtoold[82042:778426] [MT] DVTPlugInLoading: Failed to load code for plug-in com.apple.dt.AssetCatalogCocoaTouchKit (/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework), error = Error Domain=NSCocoaErrorDomain Code=3587 "dlopen_preflight(/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit): Library not loaded: /Library/Developer/PrivateFrameworks/CoreSimulator.framework/Versions/A/CoreSimulator
  Referenced from: /Applications/Xcode.app/Contents/PlugIns/IDEInterfaceBuilderCocoaTouchIntegration.framework/Versions/A/IDEInterfaceBuilderCocoaTouchIntegration
  Reason: image not found" UserInfo={NSLocalizedFailureReason=The bundle is damaged or missing necessary resources., NSLocalizedRecoverySuggestion=Try reinstalling the bundle., NSFilePath=/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit, NSDebugDescription=dlopen_preflight(/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit): Library not loaded: /Library/Developer/PrivateFrameworks/CoreSimulator.framework/Versions/A/CoreSimulator
  Referenced from: /Applications/Xcode.app/Contents/PlugIns/IDEInterfaceBuilderCocoaTouchIntegration.framework/Versions/A/IDEInterfaceBuilderCocoaTouchIntegration
  Reason: image not found, NSBundlePath=/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework, NSLocalizedDescription=The bundle “AssetCatalogCocoaTouchKit” couldn’t be loaded because it is damaged or missing necessary resources.}, dyldError = dlopen(/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit, 0): Library not loaded: /Library/Developer/PrivateFrameworks/CoreSimulator.framework/Versions/A/CoreSimulator
  Referenced from: /Applications/Xcode.app/Contents/PlugIns/IDEInterfaceBuilderCocoaTouchIntegration.framework/Versions/A/IDEInterfaceBuilderCocoaTouchIntegration
  Reason: image not found
2021-10-06 17:46:13.299 ibtoold[82042:778426] [MT] DVTPlugInExtensionFaulting: Failed to fire fault for extension AssetCatalogCocoaTouchKit.plugindata/plug-in[1]/extension[1]: Error Domain=DVTPlugInErrorDomain Code=2 "Loading a plug-in failed." UserInfo={DVTPlugInIdentifierErrorKey=com.apple.dt.AssetCatalogCocoaTouchKit, DVTPlugInExecutablePathErrorKey=/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit, NSLocalizedRecoverySuggestion=The plug-in or one of its prerequisite plug-ins may be missing or damaged and may need to be reinstalled., DVTPlugInDYLDErrorMessageErrorKey=dlopen(/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit, 0): Library not loaded: /Library/Developer/PrivateFrameworks/CoreSimulator.framework/Versions/A/CoreSimulator
  Referenced from: /Applications/Xcode.app/Contents/PlugIns/IDEInterfaceBuilderCocoaTouchIntegration.framework/Versions/A/IDEInterfaceBuilderCocoaTouchIntegration
  Reason: image not found, NSLocalizedDescription=Loading a plug-in failed., NSFilePath=/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework, NSLocalizedFailureReason=The plug-in “com.apple.dt.AssetCatalogCocoaTouchKit” at path “/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework” could not be loaded.  The plug-in or one of its prerequisite plug-ins may be missing or damaged., NSUnderlyingError=0x7f810bd869b0 {Error Domain=NSCocoaErrorDomain Code=3587 "dlopen_preflight(/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit): Library not loaded: /Library/Developer/PrivateFrameworks/CoreSimulator.framework/Versions/A/CoreSimulator
  Referenced from: /Applications/Xcode.app/Contents/PlugIns/IDEInterfaceBuilderCocoaTouchIntegration.framework/Versions/A/IDEInterfaceBuilderCocoaTouchIntegration
  Reason: image not found" UserInfo={NSLocalizedFailureReason=The bundle is damaged or missing necessary resources., NSLocalizedRecoverySuggestion=Try reinstalling the bundle., NSFilePath=/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit, NSDebugDescription=dlopen_preflight(/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit): Library not loaded: /Library/Developer/PrivateFrameworks/CoreSimulator.framework/Versions/A/CoreSimulator
  Referenced from: /Applications/Xcode.app/Contents/PlugIns/IDEInterfaceBuilderCocoaTouchIntegration.framework/Versions/A/IDEInterfaceBuilderCocoaTouchIntegration
  Reason: image not found, NSBundlePath=/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework, NSLocalizedDescription=The bundle “AssetCatalogCocoaTouchKit” couldn’t be loaded because it is damaged or missing necessary resources.}}}
Uncaught exception: NSInternalInconsistencyException Error getting value for key 'initializationClass' of extension 'AssetCatalogCocoaTouchKit.plugindata/plug-in[1]/extension[1]' in plug-in 'com.apple.dt.AssetCatalogCocoaTouchKit': Error Domain=DVTPlugInErrorDomain Code=2 "Loading a plug-in failed." UserInfo={DVTPlugInIdentifierErrorKey=com.apple.dt.AssetCatalogCocoaTouchKit, DVTPlugInExecutablePathErrorKey=/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit, NSLocalizedRecoverySuggestion=The plug-in or one of its prerequisite plug-ins may be missing or damaged and may need to be reinstalled., DVTPlugInDYLDErrorMessageErrorKey=dlopen(/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit, 0): Library not loaded: /Library/Developer/PrivateFrameworks/CoreSimulator.framework/Versions/A/CoreSimulator
  Referenced from: /Applications/Xcode.app/Contents/PlugIns/IDEInterfaceBuilderCocoaTouchIntegration.framework/Versions/A/IDEInterfaceBuilderCocoaTouchIntegration
  Reason: image not found, NSLocalizedDescription=Loading a plug-in failed., NSFilePath=/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework, NSLocalizedFailureReason=The plug-in “com.apple.dt.AssetCatalogCocoaTouchKit” at path “/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework” could not be loaded.  The plug-in or one of its prerequisite plug-ins may be missing or damaged., NSUnderlyingError=0x7f810bd869b0 {Error Domain=NSCocoaErrorDomain Code=3587 "dlopen_preflight(/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit): Library not loaded: /Library/Developer/PrivateFrameworks/CoreSimulator.framework/Versions/A/CoreSimulator
  Referenced from: /Applications/Xcode.app/Contents/PlugIns/IDEInterfaceBuilderCocoaTouchIntegration.framework/Versions/A/IDEInterfaceBuilderCocoaTouchIntegration
  Reason: image not found" UserInfo={NSLocalizedFailureReason=The bundle is damaged or missing necessary resources., NSLocalizedRecoverySuggestion=Try reinstalling the bundle., NSFilePath=/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit, NSDebugDescription=dlopen_preflight(/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit): Library not loaded: /Library/Developer/PrivateFrameworks/CoreSimulator.framework/Versions/A/CoreSimulator
  Referenced from: /Applications/Xcode.app/Contents/PlugIns/IDEInterfaceBuilderCocoaTouchIntegration.framework/Versions/A/IDEInterfaceBuilderCocoaTouchIntegration
  Reason: image not found, NSBundlePath=/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework, NSLocalizedDescription=The bundle “AssetCatalogCocoaTouchKit” couldn’t be loaded because it is damaged or missing necessary resources.}}} {
    NSUnderlyingError = "Error Domain=DVTPlugInErrorDomain Code=2 \"Loading a plug-in failed.\" UserInfo={DVTPlugInIdentifierErrorKey=com.apple.dt.AssetCatalogCocoaTouchKit, DVTPlugInExecutablePathErrorKey=/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit, NSLocalizedRecoverySuggestion=The plug-in or one of its prerequisite plug-ins may be missing or damaged and may need to be reinstalled., DVTPlugInDYLDErrorMessageErrorKey=dlopen(/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit, 0): Library not loaded: /Library/Developer/PrivateFrameworks/CoreSimulator.framework/Versions/A/CoreSimulator\n  Referenced from: /Applications/Xcode.app/Contents/PlugIns/IDEInterfaceBuilderCocoaTouchIntegration.framework/Versions/A/IDEInterfaceBuilderCocoaTouchIntegration\n  Reason: image not found, NSLocalizedDescription=Loading a plug-in failed., NSFilePath=/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework, NSLocalizedFailureReason=The plug-in \U201ccom.apple.dt.AssetCatalogCocoaTouchKit\U201d at path \U201c/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework\U201d could not be loaded.  The plug-in or one of its prerequisite plug-ins may be missing or damaged., NSUnderlyingError=0x7f810bd869b0 {Error Domain=NSCocoaErrorDomain Code=3587 \"dlopen_preflight(/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit): Library not loaded: /Library/Developer/PrivateFrameworks/CoreSimulator.framework/Versions/A/CoreSimulator\n  Referenced from: /Applications/Xcode.app/Contents/PlugIns/IDEInterfaceBuilderCocoaTouchIntegration.framework/Versions/A/IDEInterfaceBuilderCocoaTouchIntegration\n  Reason: image not found\" UserInfo={NSLocalizedFailureReason=The bundle is damaged or missing necessary resources., NSLocalizedRecoverySuggestion=Try reinstalling the bundle., NSFilePath=/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit, NSDebugDescription=dlopen_preflight(/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework/AssetCatalogCocoaTouchKit): Library not loaded: /Library/Developer/PrivateFrameworks/CoreSimulator.framework/Versions/A/CoreSimulator\n  Referenced from: /Applications/Xcode.app/Contents/PlugIns/IDEInterfaceBuilderCocoaTouchIntegration.framework/Versions/A/IDEInterfaceBuilderCocoaTouchIntegration\n  Reason: image not found, NSBundlePath=/Applications/Xcode.app/Contents/Frameworks/AssetCatalogCocoaTouchKit.framework, NSLocalizedDescription=The bundle \U201cAssetCatalogCocoaTouchKit\U201d couldn\U2019t be loaded because it is damaged or missing necessary resources.}}}";
}
Backtrace:
  0   __exceptionPreprocess (in CoreFoundation)
  1   objc_exception_throw (in libobjc.A.dylib)
  2   -[DVTExtension valueForKey:error:] (in DVTFoundation)
  3   _IDEInitializeOnePlugInAndPrerequisites (in IDEFoundation)
  4   _IDEInitializePlugIns (in IDEFoundation)
  5   IDEInitialize (in IDEFoundation)
  6   IBIDEInitialize (in ibtoold)
  7   main (in ibtoold)
  8   start (in libdyld.dylib)

 (in target '' from project '')
</details>

##### Command executed
`bundle exec fastlane ios build`
<details><pre>Details</pre>
[17:39:09]: Driving the lane 'ios build' 🚀
[17:39:09]: -----------------------------
[17:39:09]: --- Step: create_keychain ---
[17:39:09]: -----------------------------
[17:39:09]: Found keychain '~/Library/Keychains/', creation skipped
[17:39:09]: If creating a new Keychain DB is required please set the `require_create` option true to cause the action to fail
[17:39:09]: $ security list-keychains -d user
[17:39:09]: ▸ "/Users/Library/Keychains/"
[17:39:09]: Found keychain '/Users/Library/Keychains/' in list-keychains, adding to search list skipped
[17:39:09]: -------------------
[17:39:09]: --- Step: match ---
[17:39:09]: -------------------

+--------------------------------+------------------------+
|                Summary for match 2.195.0                |
+--------------------------------+------------------------+
| storage_mode                   | s3                     |
| s3_bucket                      |        |
| type                           | appstore               |
| keychain_name                  |                |
| keychain_password              | ********               |
| readonly                       | true                   |
| generate_apple_certs           | true                   |
| skip_provisioning_profiles     | false                  |
| app_identifier                 | [""] |
| username                       |       |
| team_id                        |              |
| git_branch                     | master                 |
| shallow_clone                  | false                  |
| clone_branch_directly          | false                  |
| force                          | false                  |
| force_for_new_devices          | false                  |
| include_all_certificates       | false                  |
| force_for_new_certificates     | false                  |
| skip_confirmation              | false                  |
| skip_docs                      | false                  |
| platform                       | ios                    |
| derive_catalyst_app_identifier | false                  |
| fail_on_name_taken             | false                  |
| skip_certificate_matching      | false                  |
| skip_set_partition_list        | false                  |
| verbose                        | false                  |
+--------------------------------+------------------------+

[17:39:09]: 🔓  Successfully decrypted certificates repo
[17:39:09]: Installing certificate...

+-------------------+--------------------------------------------+
|                     Installed Certificate                      |
+-------------------+--------------------------------------------+
| User ID           |                                  |
| Common Name       | Apple Distribution: ) |
| Organisation Unit |                                  |
| Organisation      |                                  |
| Country           |                                          |
| Start Datetime    |                     |
| End Datetime      |                     |
+-------------------+--------------------------------------------+

[17:39:09]: Installing provisioning profile...

+---------------------+-----------------------------------------------+---------------------------------------------------------------+
|                                                   Installed Provisioning Profile                                                    |
+---------------------+-----------------------------------------------+---------------------------------------------------------------+
| Parameter           | Environment Variable                          | Value                                                         |
+---------------------+-----------------------------------------------+---------------------------------------------------------------+
| App Identifier      |                                               |                                             |
| Type                |                                               | appstore                                                      |
| Platform            |                                               | ios                                                           |
| Profile UUID        | __appstore              | 1                          |
| Profile Name        | __appstore_profile-name | match AppStore                    |
| Profile Path        | __appstore_profile-path | /Users/Library/MobileDevice/Provisioning             |
|                     |                                               | Profiles/1.mobileprovision |
| Development Team ID | __appstore_team-id      |                                                     |
+---------------------+-----------------------------------------------+---------------------------------------------------------------+

[17:39:09]: All required keys, certificates and provisioning profiles are installed 🙌
[17:39:09]: Setting Provisioning Profile type to 'app-store'
[17:39:09]: -----------------------
[17:39:09]: --- Step: cocoapods ---
[17:39:09]: -----------------------
[17:39:09]: $ bundle exec pod install
[17:39:12]: ▸ Auto-linking React Native modules for target ``: RNAnalytics, RNAnalyticsIntegration-Adjust, RNCAsyncStorage, RNCCheckbox, RNCClipboard, RNCMaskedView, RNDeviceInfo, RNGestureHandler, RNLocalize, RNPermissions, RNReanimated, RNSVG, RNScreens, RNVectorIcons, RealmJS, react-native-background-timer, react-native-ble-plx, react-native-bluetooth-status, react-native-camera, react-native-config, react-native-geolocation-service, react-native-image-picker, react-native-mapbox-gl, react-native-maps, react-native-netinfo, react-native-safe-area-context, react-native-simple-toast, react-native-splash-screen, and react-native-webview
[17:39:13]: ▸ Analyzing dependencies
[17:39:17]: ▸ Downloading dependencies
[17:39:17]: ▸ Generating Pods project
[17:39:17]: ▸ Integrating client project
[17:39:17]: ▸ Pod installation complete! There are 65 dependencies from the Podfile and 72 total pods installed.
[17:39:17]: --------------------------------------
[17:39:17]: --- Step: increment_version_number ---
[17:39:17]: --------------------------------------
[17:39:17]: $ cd /Users/mobile-app/ios && agvtool new-marketing-version 2.5.2
[17:39:17]: ▸ Setting CFBundleShortVersionString of project  to:
[17:39:17]: ▸ 2.5.2.
[17:39:17]: ▸ Updating CFBundleShortVersionString in Info.plist(s)...
[17:39:17]: ▸ Updated CFBundleShortVersionString in ".xcodeproj/../-tvOS/Info.plist" to 2.5.2
[17:39:17]: ▸ Updated CFBundleShortVersionString in ".xcodeproj/../-tvOSTests/Info.plist" to 2.5.2
[17:39:17]: ▸ Updated CFBundleShortVersionString in ".xcodeproj/..//Info.plist" to 2.5.2
[17:39:17]: ▸ Updated CFBundleShortVersionString in ".xcodeproj/../Tests/Info.plist" to 2.5.2
[17:39:17]: ------------------------------------
[17:39:17]: --- Step: app_store_build_number ---
[17:39:17]: ------------------------------------
[17:39:17]: Login to App Store Connect (@.)
[17:39:18]: Login successful
[17:39:22]: Fetching the latest build number for live-version
[17:39:23]: Latest upload for live-version 2.52.0 is build: 2
[17:39:23]: ------------------------------------
[17:39:23]: --- Step: increment_build_number ---
[17:39:23]: ------------------------------------
Current version of project  is: 
    3

/Users/mobile-app/ios
[17:39:23]: $ cd /Users/mobile-app/ios && agvtool new-version -all 3 && cd -
[17:39:23]: ▸ Setting version of project  to:
[17:39:23]: ▸ 3.
[17:39:23]: ▸ Also setting CFBundleVersion key (assuming it exists)
[17:39:23]: ▸ Updating CFBundleVersion in Info.plist(s)...
[17:39:23]: ▸ Updated CFBundleVersion in ".xcodeproj/../-tvOS/Info.plist" to 3
[17:39:23]: ▸ Updated CFBundleVersion in ".xcodeproj/../-tvOSTests/Info.plist" to 3
[17:39:23]: ▸ Updated CFBundleVersion in ".xcodeproj/..//Info.plist" to 3
[17:39:23]: ▸ Updated CFBundleVersion in ".xcodeproj/../Tests/Info.plist" to 3
[17:39:23]: ▸ /Users/mobile-app/ios
[17:39:23]: -----------------------------------
[17:39:23]: --- Step: clean_build_artifacts ---
[17:39:23]: -----------------------------------
[17:39:23]: Cleaned up build artifacts 🐙
[17:39:23]: ---------------------------------
[17:39:23]: -----------------------
[17:39:23]: --- Step: build_app ---
[17:39:23]: -----------------------
[17:39:23]: Resolving Swift Package Manager dependencies...
[17:39:23]: $ xcodebuild -resolvePackageDependencies -workspace .xcworkspace -scheme 
[17:39:23]: ▸ Command line invocation:
[17:39:23]: ▸     /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -resolvePackageDependencies -workspace .xcworkspace -scheme 
[17:39:23]: ▸ User defaults from command line:
[17:39:23]: ▸     IDEPackageSupportUseBuiltinSCM = YES
[17:39:24]: ▸ --- xcodebuild: WARNING: Using the first of multiple matching destinations:
[17:39:24]: ▸ { platform:iOS, id:dvtdevice-DVTiPhonePlaceholder-iphoneos:placeholder, name:Any iOS Device }
[17:39:24]: ▸ { platform:iOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder, name:Any iOS Simulator Device }
[17:39:24]: ▸ resolved source packages: 
[17:39:24]: $ xcodebuild -showBuildSettings -workspace .xcworkspace -scheme 
[17:39:26]: Detected provisioning profile mapping: {:""=>"match AppStore  "}

+--------------------------------------------------------+-------------------------------------------------------------+
|                                               Summary for gym 2.195.0                                                |
+--------------------------------------------------------+-------------------------------------------------------------+
| skip_codesigning                                       | true                                                        |
| scheme                                                 |                                                  |
| clean                                                  | true                                                        |
| workspace                                              | .xcworkspace                                     |
| include_bitcode                                        | true                                                        |
| export_method                                          | app-store                                                   |
| export_options.provisioningProfiles. | match AppStore                  |
| output_directory                                       | .                                                           |
| output_name                                            |                                                         |
| silent                                                 | false                                                       |
| skip_package_ipa                                       | false                                                       |
| skip_package_pkg                                       | false                                                       |
| build_path                                             | /Users/Library/Developer/Xcode/Archives/2021-10-06 |
| result_bundle                                          | false                                                       |
| buildlog_path                                          | ~/Library/Logs/gym                                          |
| destination                                            | generic/platform=iOS                                        |
| skip_profile_detection                                 | false                                                       |
| skip_package_dependencies_resolution                   | false                                                       |
| disable_package_automatic_updates                      | false                                                       |
| use_system_scm                                         | false                                                       |
| xcode_path                                             | /Applications/Xcode.app                                     |
+--------------------------------------------------------+-------------------------------------------------------------+

[17:39:26]: $ set -o pipefail && xcodebuild -workspace .xcworkspace -scheme  -destination 'generic/platform=iOS' -archivePath /Users/Library/Developer/Xcode/Archives/2021-10-06/\ 2021-10-06\ 17.39.26.xcarchive clean archive CODE_SIGN_IDENTITY='' CODE_SIGNING_REQUIRED=NO CODE_SIGN_ENTITLEMENTS='' CODE_SIGNING_ALLOWED=NO | tee /Users/Library/Logs/gym/-.log | xcpretty
[17:39:28]: ▸ Clean Succeeded
[17:39:34]: ▸ ❌  error: failed to read asset tags: The command `(cd /Users/mobile-app/ios && /Applications/Xcode.app/Contents/Developer/usr/bin/actool --print-asset-tag-combinations --output-format xml1 /Users/mobile-app/ios//Images.xcassets)` exited with status 255. The command's standard error was:
[17:39:34]: ▸ ** ARCHIVE FAILED **
▸ Clean Succeeded

❌  error: failed to read asset tags: The command `(cd /Users/mobile-app/ios && /Applications/Xcode.app/Contents/Developer/usr/bin/actool --print-asset-tag-combinations --output-format xml1 /Users/mobile-app/ios//Images.xcassets)` exited with status 255. The command's standard error was:

** ARCHIVE FAILED **
[17:39:34]: Exit status: 65

+---------------+-------------------------+
|            Build environment            |
+---------------+-------------------------+
| xcode_path    | /Applications/Xcode.app |
| gym_version   | 2.195.0                 |
| export_method | app-store               |
| sdk           | iPhoneOS15.0.sdk        |
+---------------+-------------------------+

[17:39:34]: ▸   6   IBIDEInitialize (in ibtoold)
[17:39:34]: ▸   7   main (in ibtoold)
[17:39:34]: ▸   8   start (in libdyld.dylib)
[17:39:34]: ▸  (in target '' from project '')
[17:39:34]: 
[17:39:34]: ⬆️  Check out the few lines of raw `xcodebuild` output above for potential hints on how to solve this error
[17:39:34]: 📋  For the complete and more detailed error log, check the full log at:
[17:39:34]: 📋  /Users/Library/Logs/gym/-.log
[17:39:34]: 
[17:39:34]: Looks like  ran into a build/archive error with your project
[17:39:34]: It's hard to tell what's causing the error, so we wrote some guides on how
[17:39:34]: to troubleshoot build and signing issues: https://docs..tools/codesigning/getting-started/
[17:39:34]: Before submitting an issue on GitHub, please follow the guide above and make
[17:39:34]: sure your project is set up correctly.
[17:39:34]:  uses `xcodebuild` commands to generate your binary, you can see the
[17:39:34]: the full commands printed out in yellow in the above log.
[17:39:34]: Make sure to inspect the output above, as usually you'll find more error information there
[17:39:34]: 
+------------------------------------+------------------------------------------------------------------------+
|                                                Lane Context                                                 |
+------------------------------------+------------------------------------------------------------------------+
| PLATFORM_NAME                      | ios                                                                    |
| LANE_NAME                          | ios build                                                              |
| KEYCHAIN_PATH                      | ~/Library/Keychains/                                           |
| ORIGINAL_DEFAULT_KEYCHAIN          | "/Users/Library/Keychains/"                        |
| _PROFILE_TYPE                  | app-store                                                              |
| MATCH_PROVISIONING_PROFILE_MAPPING | {""=>"match AppStore  "} |
| VERSION_NUMBER                     | 2.5.2                                                                  |
| LATEST_BUILD_NUMBER                | 2                                                                      |
| LATEST_VERSION                     | 2.52.0                                                                 |
| BUILD_NUMBER                       | 3                                                                      |
| _PROFILE_PATHS                 |                                                                        |
| DSYM_PATHS                         |                                                                        |
| DSYM_LATEST_UPLOADED_DATE          |                                                                        |
+------------------------------------+------------------------------------------------------------------------+
[17:39:34]: Error building the application - see the log above

</details>

### Environment

swift-driver version: 1.26.9 <details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 11.6                                                           |
| Ruby                        | 2.6.3                                                          |
| Bundler?                    | true                                                           |
| Git                         | git version 2.30.1 (Apple Git-130)                             |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | macOS 11.6 (20G165)                                            |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib |
| OpenSSL Version             | LibreSSL 2.8.3                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 13.0                                                           |
| Swift Version               | 5.5                                                            |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
platform :ios do
  before_all do
    # create a new keychain
    password = SecureRandom.base64
    keychain_name = "fastlane"
    ENV["MATCH_KEYCHAIN_NAME"] = keychain_name
    ENV["MATCH_KEYCHAIN_PASSWORD"] = password
    ENV["AWS_REGION"] = ""
    create_keychain(
      name: keychain_name,
      default_keychain: true,
      unlock: true,
      timeout: 3600,
      lock_when_sleeps: true,
      password: password
    )

    # fetch provisioning profile
    match(
      storage_mode: "s3",
      s3_bucket: "",
      type: "appstore",
      keychain_name: keychain_name,
      keychain_password: password,
      readonly: true
    )
    cocoapods
  end

  lane :build do
    current_version = "2.5.2"
    increment_version_number(
      version_number: "#{current_version}", 
      xcodeproj: ""
    )
    increment_build_number(
      build_number: app_store_build_number + 1,
      xcodeproj: ""
    )
    clean_build_artifacts
    commit_version_bump(
      force: true,
      xcodeproj: "j"
    )
    build_app(skip_codesigning: true, scheme: "", clean: true,  workspace: "", include_bitcode: true)
  end
end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier("") # The bundle identifier of your app
apple_id("") # Your Apple email address
team_id("")

# For more information about the Appfile, see:
#     https://docs.fastlane.tools/advanced/#appfile

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.195.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                             | Version      |
| ------------------------------- | ------------ |
| did_you_mean                    | 1.3.0        |
| bundler                         | 2.2.28       |
| rake                            | 13.0.6       |
| rexml                           | 3.2.5        |
| CFPropertyList                  | 3.0.4        |
| concurrent-ruby                 | 1.1.9        |
| i18n                            | 1.8.10       |
| minitest                        | 5.14.4       |
| tzinfo                          | 2.0.4        |
| zeitwerk                        | 2.4.2        |
| activesupport                   | 6.1.4.1      |
| public_suffix                   | 4.0.6        |
| addressable                     | 2.8.0        |
| httpclient                      | 2.8.3        |
| json                            | 2.5.1        |
| algoliasearch                   | 1.27.5       |
| artifactory                     | 3.0.15       |
| atomos                          | 0.1.3        |
| aws-eventstream                 | 1.2.0        |
| aws-partitions                  | 1.510.0      |
| aws-sigv4                       | 1.4.0        |
| jmespath                        | 1.4.0        |
| aws-sdk-core                    | 3.121.1      |
| aws-sdk-kms                     | 1.49.0       |
| aws-sdk-s3                      | 1.103.0      |
| babosa                          | 1.0.4        |
| claide                          | 1.0.3        |
| fuzzy_match                     | 2.0.4        |
| nap                             | 1.1.0        |
| netrc                           | 0.11.0       |
| ffi                             | 1.15.4       |
| ethon                           | 0.14.0       |
| typhoeus                        | 1.4.0        |
| cocoapods-core                  | 1.11.2       |
| cocoapods-deintegrate           | 1.0.5        |
| cocoapods-downloader            | 1.5.1        |
| cocoapods-plugins               | 1.0.0        |
| cocoapods-search                | 1.0.1        |
| cocoapods-trunk                 | 1.6.0        |
| cocoapods-try                   | 1.2.0        |
| colored2                        | 3.1.2        |
| escape                          | 0.0.4        |
| fourflusher                     | 2.3.1        |
| gh_inspector                    | 1.1.3        |
| molinillo                       | 0.8.0        |
| ruby-macho                      | 2.5.1        |
| nanaimo                         | 0.3.0        |
| xcodeproj                       | 1.21.0       |
| cocoapods                       | 1.11.2       |
| colored                         | 1.2          |
| highline                        | 2.0.3        |
| commander                       | 4.6.0        |
| declarative                     | 0.0.20       |
| digest-crc                      | 0.6.4        |
| unf_ext                         | 0.0.8        |
| unf                             | 0.1.4        |
| domain_name                     | 0.5.20190701 |
| dotenv                          | 2.7.6        |
| emoji_regex                     | 3.2.3        |
| excon                           | 0.86.0       |
| faraday-em_http                 | 1.0.0        |
| faraday-em_synchrony            | 1.0.0        |
| faraday-excon                   | 1.1.0        |
| faraday-httpclient              | 1.0.1        |
| faraday-net_http                | 1.0.1        |
| faraday-net_http_persistent     | 1.2.0        |
| faraday-patron                  | 1.0.0        |
| faraday-rack                    | 1.0.0        |
| multipart-post                  | 2.0.0        |
| ruby2_keywords                  | 0.0.5        |
| faraday                         | 1.8.0        |
| http-cookie                     | 1.0.4        |
| faraday-cookie_jar              | 0.0.7        |
| faraday_middleware              | 1.1.0        |
| fastimage                       | 2.2.5        |
| jwt                             | 2.3.0        |
| memoist                         | 0.16.2       |
| multi_json                      | 1.15.0       |
| os                              | 1.1.1        |
| signet                          | 0.16.0       |
| googleauth                      | 1.0.0        |
| mini_mime                       | 1.1.1        |
| trailblazer-option              | 0.1.1        |
| uber                            | 0.1.0        |
| representable                   | 3.1.1        |
| retriable                       | 3.1.2        |
| webrick                         | 1.7.0        |
| google-apis-core                | 0.4.1        |
| google-apis-androidpublisher_v3 | 0.11.0       |
| google-apis-playcustomapp_v1    | 0.5.0        |
| google-apis-iamcredentials_v1   | 0.7.0        |
| google-apis-storage_v1          | 0.8.0        |
| google-cloud-env                | 1.5.0        |
| google-cloud-errors             | 1.2.0        |
| google-cloud-core               | 1.6.0        |
| google-cloud-storage            | 1.34.1       |
| mini_magick                     | 4.11.0       |
| naturally                       | 2.2.1        |
| optparse                        | 0.1.1        |
| plist                           | 3.6.0        |
| rubyzip                         | 2.3.2        |
| security                        | 0.1.3        |
| simctl                          | 1.6.8        |
| terminal-notifier               | 2.0.0        |
| unicode-display_width           | 1.8.0        |
| terminal-table                  | 1.8.0        |
| tty-screen                      | 0.8.1        |
| tty-cursor                      | 0.7.1        |
| tty-spinner                     | 0.9.3        |
| word_wrap                       | 1.0.0        |
| rouge                           | 2.0.7        |
| xcpretty                        | 0.3.0        |
| xcpretty-travis-formatter       | 1.0.1        |
</details>

*generated on:* **2021-10-06**
</details>

<details>
  <pre> [REPLACE THIS WITH YOUR INFORMATION] </pre>
</details>
